src/sql/workbench/contrib/profiler/browser/profilerEditor.ts

```diff
@@ -501,6 +501,12 @@ export class ProfilerEditor extends EditorPane {
 			if (savedViewState) {
 				this._profilerTableEditor.restoreViewState(savedViewState);
 			}
+
+			if (this.input.isFileSession && this.input.isSetupPhase) {		// Add loading indicator when opening a new file session
+				this._profilerTableEditor.loadingSpinner.loading = true;
+				this._profilerTableEditor.loadingSpinner.loadingMessage = nls.localize('Loading', "Loading...");
```

Comment:
if this doesn't get set, does the default loading message get used? https://github.com/microsoft/azuredatastudio/blob/main/src/sql/base/browser/ui/loadingSpinner/loadingSpinner.component.ts#L11

Reply:
I didn't realize that the custom message was not getting used here. Removed it.